docs/experience/breakdown.md

```diff
@@ -0,0 +1,30 @@
+title: Sematext Experience Detailed Timing Breakdown
+description: Investigate page loads and HTTP Requests performance breakdown with Sematext Experience
```

Comment:
description: Investigate page loads and HTTP Requests performance breakdown with Sematext Experience
=>
description: Investigate page-load and HTTP request performance with the detailed timing breakdown in Sematext Experience